### DISCUSSION TITLE
GitHub student pack verification follow-up

### DISCUSSION THREAD

**Discussion by Paravk** [community/community#81425] [GitHub Education]
### Hi everyone,

Hi everyone I applied for GitHub student pack yesterday through college mail and i got approved it is showing" **Congrats! Your request for [@Paravk](https://github.com/Paravk) was approved and your benefits will be available soon.**" when will I be able to access the benifits and how will I be able to access them ,like will I get a mail??
![GitHub approval message](https://github.com/community/community/assets/154521547/361732a3-557c-44b6-8d34-5bd0a9191295)

**Comment by n-npc** [comment]
Im with the same problem "You were last verified as a student on Dec 20, 2023."... It has been 6 days, I mean I'm not complaining that much, is just anxiety to see if there was something wrong or something
